### PR TITLE
update mapper doc

### PR DESCRIPTION
**Pull request by nilisha-jais**
Signed-off-by: Nilisha Jaiswal <jaiswal.nilisha05@gmail.com>

* **Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

**Review by fisherxu — COMMENTED** [review]
cc @RyanZhaoXB Please take a look at for the content :)

File: content/en/docs/developer/mappers/_index.md [hunk 1]
```diff
@@ -10,7 +10,24 @@ toc: true
 type: docs
 ---
 
+## Overview
+Mapper is an application that is used to connect and control devices. Following are the responsibilities of mapper:
```
Comment (RyanZhaoXB) [inline content/en/docs/developer/mappers/_index.md:14]: “Following are the responsibilities of mapper” seems a little weird. Maybe "The responsibilities of mapper are as follows" looks better.

File: content/en/docs/developer/mappers/_index.md [hunk 2]
```diff
@@ -10,7 +10,24 @@ toc: true
...
 
-- [bluetooth](./bluetooth)
+- Scan and connect to the device.
+- Report the actual state of twin-attributes of device.
+- Map the expected state of device-twin to actual state of device-twin.
+- Collect telemetry data from device.
+- Convert readings from device to format accepted by KubeEdge.
+- Schedule actions on the device.
```
Comment (RyanZhaoXB) [inline content/en/docs/developer/mappers/_index.md:21]: "Schedule actions on the device." What function does this item refer to?

Comment (nilisha-jais) [reply content/en/docs/developer/mappers/_index.md:21]: The function is referring to the [scheduler](https://kubeedge.io/en/docs/developer/mappers/bluetooth/#scheduler) module.

File: content/en/docs/developer/mappers/_index.md (hunk 1 above)
Comment (nilisha-jais) [reply content/en/docs/developer/mappers/_index.md]: Updated it!

File: content/en/docs/developer/mappers/_index.md (hunk 2 above)
Comment (RyanZhaoXB) [reply content/en/docs/developer/mappers/_index.md:21]: Thank you. lgtm :)

**Pull request merged**